draft-demarco-oauth-status-attestations.md

```diff
@@ -361,6 +366,32 @@ Below a non-normative example of an HTTP Response with an error.
   }
 ~~~
 
+Since the Status Assertion Request is an array of strings, if one or more status
+assertion request are made for a Digital Credential with errors,
```

Comment:
```suggestion
Since the Status Assertion Request is an array of strings, if one or more Status
Assertion Request object are made for a Digital Credential with errors,
```